source/core/src/main/com/csse3200/game/components/player/PlayerInventoryDisplay.java

```diff
@@ -34,9 +34,11 @@ public class PlayerInventoryDisplay extends UIComponent {
     private boolean toggle = false; // Whether inventory is toggled on;
     DialogueBox itemOverlay;
 
+    private boolean toggleHotbar = true;
```

Comment:
Is there a reason we need a toggleHotbar, should this not just always be the opposite of toggle?